src/scribe_data/cli/main.py

```diff
@@ -159,6 +160,12 @@ def main() -> None:
         default="camel",
         help="The case format for identifiers in the output data (default: camel).",
     )
+    get_parser.add_argument(
+        "-wd",
+        "--wikidata-dump",
```

Comment:
```suggestion
        "--wikidata-dump-path",
```
I'm wondering if this should be something like the above to specify it's a local path to an existing dump? Thoughts? Or maybe `--local-wikidata-dump`?